File: Logistics.Web/ClientApp/src/app/models/hand-receipt.ts
import { HandReceiptItem } from './hand-receipt-item';
import { HandReceiptVerification } from './hand-receipt-verification';
import { Site } from './site';
import { User } from './user';

export class HandReceipt {
  id: number;
  siteId: number;
  userId: number;
  remarks: string;
  dateCreated: Date;
  site: Site;
  user: User;
  handReceiptItems: HandReceiptItem[];
  handReceiptVerifications: HandReceiptVerification[];
}

File: Logistics.Data/HandReceipt.cs
using System;
using System.Collections.Generic;

namespace Logistics.Data
{
    public class HandReceipt
    {
        public int Id { get; set; }
        public int SiteId { get; set; }
        public int UserId { get; set; }
        public string Remarks { get; set; }
        public DateTime DateCreated { get; set; }

        public Site Site { get; set; }
        public User User { get; set; }

        public List<HandReceiptItem> HandReceiptItems { get; set; }
        public List<HandReceiptVerification> HandReceiptVerifications { get; set; }
    }
}
